Cargo.toml

```diff
@@ -62,6 +62,7 @@ rustversion = "1.0.7"
 serde_json = "1.0.68"
 serde_plain = "1.0.0"
 surf = { version = "2.3.1", optional = true }
+uuid = { version = "0.8.2", features = ["v4"] }
```

Comment:
highest version running on rustc 1.60.0. I also considered embedding uuid generation function into the library itself but my first option is to use a lib.